src/main/controllers/ClaimantNoticePeriodController.ts

```diff
@@ -30,12 +28,12 @@ export default class ClaimantNoticePeriodController {
             label: (l: AnyRecord): string => l.no,
             value: YesOrNoOrNotSure.NO,
             subFields: {
-              whatAreClaimantCorrectNoticeDetails: {
-                type: 'textarea',
-                id: 'whatAreClaimantCorrectNoticeDetails',
-                label: (l: AnyRecord): string => l.whatAreClaimantCorrectNoticeDetails.label,
+              et3ResponseCorrectNoticeDetails: {
+                type: 'charactercount',
+                label: (l: AnyRecord): string => l.et3ResponseCorrectNoticeDetails.label,
                 labelSize: 's',
-                hint: (l: AnyRecord): string => l.whatAreClaimantCorrectNoticeDetails.hint,
+                hint: (l: AnyRecord): string => l.et3ResponseCorrectNoticeDetails.hint,
+                maxlength: 2500,
```

Comment:
Can we have this set to 500 please? This box is smaller and cut the text if we allow more than 500